system_tests/bigtable.py

```diff
@@ -356,6 +356,47 @@ def _write_to_row(self, row1=None, row2=None, row3=None, row4=None):
         cell4 = Cell(CELL_VAL4, timestamp4)
         return cell1, cell2, cell3, cell4
 
+    def test_read_large_cell_limit(self):
+        row = self._table.row(ROW_KEY)
+        self.rows_to_delete.append(row)
+
+        number_of_bytes = 10 * 1024 * 1024
+        data = '1' * number_of_bytes  # 10MB of 1's.
+        row.set_cell(COLUMN_FAMILY_ID1, COL_NAME1, data)
+        row.commit()
+
+        # Read back the contents of the row.
+        partial_row_data = self._table.read_row(ROW_KEY)
+        self.assertEqual(partial_row_data.row_key, ROW_KEY)
+        cell = partial_row_data.cells[COLUMN_FAMILY_ID1]
+        column = cell[COL_NAME1]
+        value = column[0].value
+        self.assertIsNotNone(value)
+        self.assertEqual(len(value), number_of_bytes)
+
+    def test_read_large_cells_over_limit(self):
+        row = self._table.row(ROW_KEY)
+        self.rows_to_delete.append(row)
+
+        data = '1' * 51 * 1024 * 1024  # 51MB.
+        row.set_cell(COLUMN_FAMILY_ID1, COL_NAME1, data)
+        row.commit()
+        row.set_cell(COLUMN_FAMILY_ID1, COL_NAME1, data)
+        row.commit()
+
+        with self.assertRaises(Exception):
+            partial_row_data = self._table.read_row(ROW_KEY)
+
+
+    def test_read_large_cell_over_limit(self):
+        row = self._table.row(ROW_KEY)
+        self.rows_to_delete.append(row)
+
+        data = '1' * 101 * 1024 * 1024  # 11MB of 1's.
+        row.set_cell(COLUMN_FAMILY_ID1, COL_NAME1, data)
+        with self.assertRaises(Exception):
```

Comment:
It throws a `_RENDEZVOUS` exception.

So the request is made but then I believe upstream doesn't send the data back since the header says "you can only have 100MB!". At least that's how I think it works.